src/Test/DeployCoreClrTestRuntime/DeployCoreClrTestRuntime.csproj

```diff
@@ -10,9 +10,10 @@
     <Prefer32Bit>false</Prefer32Bit>
     <LargeAddressAware>true</LargeAddressAware>
     <TargetFramework>netcoreapp2.0</TargetFramework>
-    <RuntimeIdentifiers>win7-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;osx.10.12-x64</RuntimeIdentifiers>
     <PackageTargetFallback>portable-net452;dotnet;netstandard1.6</PackageTargetFallback>
-    <SelfContained>true</SelfContained>
+    <RuntimeIdentifiers>win7-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;osx.10.12-x64</RuntimeIdentifiers>
+    <RuntimeIdentifier>$(RoslynRuntimeIdentifier)</RuntimeIdentifier>
+    <SelfContained Condition="'$(RoslynRuntimeIdentifier)' != ''" >true</SelfContained>
```

Comment:
Not sure I understand the root cause here -- does SelfContained not work on older builds? Wouldn't always setting it also break things on dev machines?